spec/spec_helper.rb

```diff
@@ -64,6 +64,15 @@
   end
 }
 
+def sensuctl_types
```

Comment:
what's going on here?

Reply:
This mimics logic in `sensuctl_config` type that finds all types that have provider `sensuctl` since that's the ones we want to deal with.  Rather than hardcode the list of types this finds them dynamically.

Reply:
could you please document that in the code

Reply:
Documented in code